active-fedora.gemspec

```diff
@@ -8,11 +9,11 @@ Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.authors     = ["Matt Zumwalt", "McClain Looney", "Justin Coyne"]
   s.email       = ["samvera-tech@googlegroups.com"]
-  s.homepage    = %q{https://github.com/samvera/active_fedora}
-  s.summary     = %q{A convenience libary for manipulating documents in the Fedora Repository.}
-  s.description = %q{ActiveFedora provides for creating and managing objects in the Fedora Repository Architecture.}
+  s.homepage    = 'https://github.com/samvera/active_fedora'
+  s.summary     = 'A convenience libary for manipulating documents in the Fedora Repository.'
+  s.description = 'ActiveFedora provides for creating and managing objects in the Fedora Repository Architecture.'
   s.license = "Apache-2.0"
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.6'
```

Comment:
Should this match the minimum tested ruby version?

Reply:
I'm not sure it really needs to, but I could add testing back in for ruby 2.6.